dev-haskell/hlint/hlint-1.8.57.ebuild

```diff
@@ -23,6 +23,7 @@ RDEPEND=">=dev-haskell/cpphs-1.11:=[profile?]
 	>=dev-haskell/transformers-0.0:=[profile?]
 	>=dev-haskell/uniplate-1.5:=[profile?]
 	>=dev-lang/ghc-6.10.4:=
+	emacs? ( >=app-editors/emacs-23.1:* )
```

Comment:
Unless my eyes deceive me, you're not removing anything here.

Reply:
Right, that one was missing the dependency before. It certainly needs it, because it is calling elisp-compile.

I'm going to update the commit message.